infrastructure/outputs.tf

```diff
@@ -22,4 +22,27 @@ output "cluster_name" {
 
 output "cluster_region" {
   value = var.cluster_region
-}
\ No newline at end of file
+}
+
+output "endpoint" {
+  value = var.create_cluster && var.autopilot_cluster && var.private_cluster ? module.private-gke-autopilot-cluster[0].endpoint : (
```

Comment:
can these be defined in main.tf as locals instead? this is quite complex.

maybe we can have locals for: 1) public standard 2) public AP 3) private standard 4) private AP